### PR TITLE
`dl-v2` cli improvements 2

### PR DESCRIPTION
**Pull request by aniketinfocusp**
This PR includes
- confirmation messages before modifying commands (there is option to bypass that with `-y`
- ordering table fields.

### CODE REVIEW

File: weather_dl_v2/cli/app/subcommands/license.py [hunk 1]
```diff
@@ -26,6 +26,11 @@ class LicenseValidator(Validator):
     pass
 
 
+license_key_order = [
+    'secret_id', 'k8s_deployment_id', 'status', 'client_name', 'license_id', 'number_of_requests'
```
Comment (mahrsee1997) [inline weather_dl_v2/cli/app/subcommands/license.py:30]: nit: license_id, client_name, status, number_of_requests, secret_id, k8s_deployment_id.

File: weather_dl_v2/cli/app/subcommands/queue.py [hunk 2]
```diff
@@ -26,6 +26,9 @@ class QueueValidator(Validator):
     pass
 
 
+queue_key_order = ['client_name', 'license_id', 'queue']
```
Comment (mahrsee1997) [inline weather_dl_v2/cli/app/subcommands/queue.py:29]: nit: license_id, client_name, queue

**Review by mahrsee1997 — APPROVED** [review]
Few nits rest LGTM.

I would love to see follow PR for:
1) Updating readme.md file.
2) As discussed offline, in case of `license redeploy` if user not passing any inputs then show appropriate error message on CLI + find other end-points with sam issue & fix that as well.

File: weather_dl_v2/cli/app/subcommands/queue.py (hunk 2 above)
Comment (aniketinfocusp) [reply weather_dl_v2/cli/app/subcommands/queue.py]: updated in [7887b24](https://github.com/google/weather-tools/pull/424/commits/7887b243c8daf71792e56390c2f561f307bf24f0)

File: weather_dl_v2/cli/app/subcommands/license.py (hunk 1 above)
Comment (aniketinfocusp) [reply weather_dl_v2/cli/app/subcommands/license.py]: updated in [7887b24](https://github.com/google/weather-tools/pull/424/commits/7887b243c8daf71792e56390c2f561f307bf24f0)

**Pull request merged**